### ISSUE TITLE
Missing dependency during build process

### ISSUE THREAD

**Issue by lguima** [ptkdev-boilerplate/node-cli-boilerplate#1]
### Versions

-   **App Version:** v2.0.2
-   **Node Version:** v14.19.1
-   **Operating System:** Ubuntu 20.04
-   **Browser:** N/A

### Expected Behavior

Run the `npm run build` without errors.

### Actual Behavior

There is an error during the build process.

```
$ npm run build

> @ptkdev/node-cli-boilerplate@2.0.2 build /<path>
> tsc

app/translations/translate.ts:10:21 - error TS2307: Cannot find module '@configs/config' or its corresponding type declarations.

10 import configs from "@configs/config";
                       ~~~~~~~~~~~~~~~~~

app/utils/logger.ts:12:20 - error TS2307: Cannot find module '@configs/config' or its corresponding type declarations.

12 import config from "@configs/config";
                      ~~~~~~~~~~~~~~~~~


Found 2 errors.

npm ERR! code ELIFECYCLE
npm ERR! errno 2
npm ERR! @ptkdev/node-cli-boilerplate@2.0.2 build: `tsc`
npm ERR! Exit status 2
npm ERR! 
npm ERR! Failed at the @ptkdev/node-cli-boilerplate@2.0.2 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /<path>/.npm/_logs/2022-04-04T17_11_13_979Z-debug.log
```

### Steps to Reproduce

1. Run `npm install`
2. Run `npm run build`

### Evidences

```
$ npm install
npm WARN read-shrinkwrap This version of npm is compatible with lockfileVersion@1, but package-lock.json was generated for lockfileVersion@2. I'll try to do my best with it!

> nodemon@2.0.15 postinstall /<path>/node_modules/nodemon
> node bin/postinstall || exit 0


> @ptkdev/node-cli-boilerplate@2.0.2 prepare /<path>
> ts-patch install -s && husky install

husky - Git hooks installed
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.2.13 (node_modules/livereload/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.13: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@2.3.2 (node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.3.2: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})

added 1003 packages from 630 contributors in 41.758s

75 packages are looking for funding
  run `npm fund` for details
```

**Comment by ptkdev** [comment]
Thanks!
Run `npm run dev` and after `npm run build`. 

PROD:
`npm run release`

DEV:
`npm run dev`

**Comment by lguima** [comment]
Thank you, now the build worked.

But then the link failed:

```
$ npm run link

> @ptkdev/node-cli-boilerplate@2.0.2 link /<path>
> sudo npm link

[sudo] password for <user>: 
sudo: npm: command not found
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @ptkdev/node-cli-boilerplate@2.0.2 link: `sudo npm link`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @ptkdev/node-cli-boilerplate@2.0.2 link script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /<path>/.npm/_logs/2022-04-04T19_06_55_867Z-debug.log
```

**Comment by ptkdev** [comment]
On unix/mac:
`sudo npm link`

On windows:
`npm link`

I will fix the readme in the next commit.

**Comment by lguima** [comment]
The script `link` runs `sudo npm link`, then explicitly running `npm link` doesn't work either.

Thanks anyway!

**Comment by ptkdev** [comment]
Yep, misprint in the readme. I will fix this today.

**Comment by ptkdev** [comment]
Fixed in `v2.0.3`
